src/main/java/com/synopsys/integration/create/apigen/data/LinkResponseDefinitions.java

```diff
@@ -61,6 +61,10 @@ public LinkResponseDefinitions() {
         componentVersionViewDefinitions.put("origins", cvvOriginsDefinition);
         final LinkResponseDefinitionItem cvvVulnerabilitiesDefinition = new LinkResponseDefinitionItem(true, "VulnerabilityView");
         componentVersionViewDefinitions.put("vulnerabilities", cvvVulnerabilitiesDefinition);
+        final LinkResponseDefinitionItem cvvUpgradeGuidanceDefinition = new LinkResponseDefinitionItem(false, "ComponentVersionUpgradeGuidanceView");
+        componentVersionViewDefinitions.put("upgrade-guidance", cvvUpgradeGuidanceDefinition);
+        final LinkResponseDefinitionItem cvvRemediationDefinition = new LinkResponseDefinitionItem(false, "ComponentVersionRemediatingView");
+        componentVersionViewDefinitions.put("remediating", cvvRemediationDefinition);
```

Comment:
The remediating link is not being added when the generator is run for some reason.